elephant/phase_analysis.py

```diff
@@ -307,3 +307,80 @@ def phase_difference(alpha, beta):
     delta = alpha - beta
     phase_diff = np.arctan2(np.sin(delta), np.cos(delta))
     return phase_diff
+
+
+def weighted_phase_lag_index(signal_i, signal_j, sampling_frequency,
+                             absolute_value=True):
+    r"""
+    Calculates the Weigthed Phase-Lag Index (WPLI)
+
+    This function expects the two signals (each containing multiple trials).
+    It Fourier-transforms the signals and for each trial pair, it calculates
+    the cross-spectrum. With that the WPLI is calculated.
+
+    Parameters
+    ----------
+    signal_i, signal_j: (t, n) np.array
+        Time-series of the first and second signals,
+        with `t` time points and `n` trials.
+    sampling_frequency: quantity/ scaler
+        Sampling frequency of the signals in Hz.
+    absolute_value: boolean (default: True)
+        Takes the absolute value of the numerator in the WPLI-formula.
+
+    Returns
```

Comment:
It seems that the function returns `freqs` and `wpli`. Therefore, both must be described here.